benchmarks/helpers/main.js

```diff
@@ -1,11 +1,21 @@
+/* eslint-disable max-statements */
+import assert from 'assert'
 import { performance, PerformanceObserver } from 'perf_hooks'
 import { fileURLToPath } from 'url'
 
 export const FIXTURES_DIR = fileURLToPath(new URL('../fixtures', import.meta.url))
 
+const sleep = (ms) =>
+  new Promise((resolve) => {
+    setTimeout(resolve, ms)
+  })
+
 export const timeFunction = async (func, runs = 1) => {
   const finishedRuns = new Map()
 
+  performance.clearMarks()
+  performance.clearMeasures()
```

Comment:
Not strictly necessary but if we run multiple benchmarks after each other, this is critical. Anyway it doesn't hurt.